sh/test-data/helm/mongodb/templates/NOTES.txt

```diff
@@ -0,0 +1,96 @@
+This Helm chart is deprecated
```

Comment:
?

Reply:
They moved to the bitnami charts repo 2 days ago. I now use the bitnami charts repo.